scripts/game_structure/load_cat.py

```diff
@@ -250,15 +257,15 @@ def json_load():
                     else:
                         new_cat.status.send_to_afterlife(target=CatGroup.STARCLAN)
 
-                # these should properly change the cat's status to align with old bool info
-                if not new_cat.dead and cat.get("exiled"):
-                    new_cat.status.exile_from_group()
-                if (
-                    not new_cat.dead
-                    and cat.get("outside")
-                    and not new_cat.status.is_outsider
-                ):
-                    new_cat.status.become_lost()
+                else:
+                    # these should properly change the cat's status to align with old bool info
+                    if cat.get("exiled"):
+                        new_cat.status.exile_from_group()
+                    if cat.get("outside") and not new_cat.status.is_outsider:
```

Comment:
shouldn't this be `elif`? since old style saves would have `exiled` be true as well as `outside` iirc, and the previous step has just made `new_cat.status.is_outsider == True`.